### PR TITLE
Merge `dev-ru` to `main` to launch Russian localisation

### PR DESCRIPTION
**Pull request by shurup**
### Describe your changes
The Russian version of Glossary is now ready to be merged into the `main` branch.
Our team (@kirkonru & @tym83 & @shurup) has translated 24 terms (and other [minimum required content](https://github.com/cncf/glossary/blob/main/LOCALIZATION.md#3-4-minimum-required-content)) into Russian.
I have checked that the localisation pages work, by building the website locally and via the preview feature in the previous PR (#3029).
We have also prepared a blog post to announce this localisation.

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

**Review by kirkonru — APPROVED** [review]

**Comment by kirkonru** [comment]
LGTM

**Review by tym83 — APPROVED** [review]
LGTM

**Review by iamNoah1 — APPROVED** [review]

**Comment by seokho-son** [comment]
Checklist to make localization live.
- [x] validate all files which are not in the localization directory (content/ru/*)
- [x] check i18n/ru.toml exist
- [x] check l10n configuration in config.toml
- [ ] acknowledge to initial localization contributors list
- [x] check Preview works

File: content/ru/_index.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+[Chris Aniszczyk](https://www.linkedin.com/in/caniszczyk/), 
+[Daniel Jones](https://www.linkedin.com/in/danieljoneseb/?originalSubdomain=uk), 
+[Jason Morgan](https://www.linkedin.com/in/jasonmorgan2/), 
+[Katelin Ramer](https://www.linkedin.com/in/katelinramer/), 
+[Mike Foster](https://www.linkedin.com/in/mfosterche/?originalSubdomain=ca), 
+и многих других авторов. 
+Полный список участников проекта можно найти на [отдельной странице](https://github.com/cncf/glossary/graphs/contributors) на GitHub.
+
+Хранителями глоссария являются:
+- [Noah Ispas](https://www.linkedin.com/in/noah-ispas-0665b42a/);
```
Comment (seokho-son) [inline content/ru/_index.md:39]:
```suggestion
- [Seokho Son](https://www.linkedin.com/in/seokho-son/);
- [Noah Ispas](https://www.linkedin.com/in/noah-ispas-0665b42a/); 
```

File: content/ru/_index.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+Хранителями глоссария являются:
+- [Noah Ispas](https://www.linkedin.com/in/noah-ispas-0665b42a/); 
+- [Jihoon Seo](https://www.linkedin.com/in/jihoon-seo/);
+- [Nate W.](https://www.linkedin.com/in/nate-double-u/);
+- [Jorge Castro](https://www.linkedin.com/in/jorge-castro2112/).
+
+[Catherine Paganini](https://www.linkedin.com/in/catherinepaganini/en/)
+и [Jason Morgan](https://www.linkedin.com/in/jasonmorgan2/)
+являются Почетными Хранителями, и мы глубоко признательны
+за их неоценимый вклад в развитие проекта на протяжении многих лет.
```
Comment (seokho-son) [inline content/ru/_index.md:47]:
```suggestion
за их неоценимый вклад в развитие проекта на протяжении многих лет.

Lastly, the Russian localization of the Cloud Native Glossary was initiated through the contributions of the Russian Localization Team (#glossary-localization-korean), including ..., ..., ...., ..., .., ..., various other contributors.
```

Please add acknowledgement for major localization contributors who participated this first version. :)

**Review by seokho-son — COMMENTED** [review]
Hi @shurup and Russian localization team!!
Thanks for this nice work!

We need to add ack for the Russian localization team to promote them and invite more contributors.

Please check the following comment.
Also, Please open another PR to update this document(PR). Don't make direct change to this PR.

**Comment by shurup** [comment]
Thank you, @seokho-son! I proposed these changes in #3037 (not sure whether I could do it in this PR directly).

**Comment by seokho-son** [comment]
Thanks @shurup 
I will take a look again once the https://github.com/cncf/glossary/pull/3037 merged. (I don't have to approve it)

FYI, we need to avoid working on upstream branches (this PR is to merge upstream/dev-ru into upstream/main)
So, using another PR to update upstream/dev-ru is appropriate approach. :)
This PR will be updated automatically, once #3037 has been merged in to dev-ru.

**Comment by shurup** [comment]
Great, thanks for your explanation, @seokho-son! We are ready to proceed, I guess :rocket:

**Comment by shurup** [comment]
Hi @seokho-son! Now, with your suggestions being applied, can remove the `hold` label and proceed? Or is something else stopping us (e.g. more reviewers, etc.)?

**Comment by seokho-son** [comment]
Based on https://github.com/cncf/glossary/pull/3031#issuecomment-2044005020
, LGTM!!!

Thanks to all contributors!!

**Review by seokho-son — APPROVED** [review]

**Pull request merged**